components/iotserver-api/org.wso2.carbon.device.mgt.iot.common.api/src/main/java/org/wso2/carbon/device/mgt/iot/common/api/DevicesManagerService.java

```diff
@@ -613,19 +616,31 @@ public boolean setStatus(@PathParam("type") String type, @PathParam("identifier"
         }
     }
 
-    @Path("/device/type/{type}/identifier/{identifier}/claimable")
+    @Path("/device/type/{type}/identifier/{identifier}/sensor/{sensorName}")
+    @POST
+    @Consumes("application/json")
+    @Produces("application/json")
+    public boolean setSensorValue(@PathParam("type") String type, @PathParam("identifier") String deviceId,
+                                  @PathParam("sensorName") String sensorName,
+                                  @HeaderParam("sensorValue") String sensorValue){
+
+        try {
+            return SensorDataManager.getInstance().setSensorRecord(deviceId, sensorName, sensorValue, Calendar
+                    .getInstance().getTimeInMillis());
+        } finally {
+            this.endTenantFlow();
```

Comment:
No tenant flow seems to have been initiated. So, what's expected out of this particular line?